htu21d.py

```diff
@@ -43,30 +44,30 @@ def _crc_check(self, value):
 
     def _issue_measurement(self, write_address):
         """Issue a measurement.
-
         Args:
             write_address (int): address to write to
         :return:
         """
+        import time
         self.i2c.start()
         self.i2c.writeto_mem(int(self.ADDRESS), int(write_address), '')
         self.i2c.stop()
         data = bytearray(3)
+        time.sleep_ms(50)
```

Comment:
im not really sure if this is needed. i did not need it at all. Will see if `readfrom_into` is a blocking request.

Reply:
according to the datasheet (https://cdn-shop.adafruit.com/datasheets/1899_HTU21D.pdf) you should wait at least 50ms before reading the result. All arduino libraries do the same.
Can't remember if I added this because I had problems, but I probably did.